File: mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(){
    int height;
    int spaces;
    //get vaid input from user
    do{
        height = get_int("Height: ");
    }while(!(height>0 && height<=8));

    for(int i=1; i<=height; i++){
        spaces = height - i;
        //print spaces
        for(int j=spaces; j>0;j--){
            printf(" ");
        }
        //left pyramid
        for(int k=0;k<i;k++){
            printf("#");
        }
        //2 space gap
        printf("  ");
        //right pyramid
        for(int l=0;l<i;l++){
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

def main():
    while True:
        height = get_int("Height: ")
        if height>=1 and height<=8:
            build(height)
            break


def build(h):
    for i in range(1,h+1):
        space = h-i
        for j in range(space,0,-1):
            print(" ",end="")
        for k in range(1,i+1):
            print("#",end="")

        print("  ",end="")
        for l in range(1,i+1):
            print("#",end="")
        print()

main()
